great_expectations/render/renderer/slack_renderer.py

```diff
@@ -43,57 +42,39 @@ def render(  # noqa: PLR0913
 
         return blocks
 
-    def _build_title_block(
+    def _build_description_block(
         self,
         validation_result: ExpectationSuiteValidationResult,
-        show_failed_expectations: bool,
         validation_result_urls: list[str],
     ) -> dict:
-        expectation_suite_name = validation_result.suite_name
-        data_asset_name = validation_result.asset_name or "__no_data_asset_name__"
-        n_checks_succeeded = validation_result.statistics["successful_expectations"]
-        n_checks = validation_result.statistics["evaluated_expectations"]
-        run_id = validation_result.meta.get("run_id", "__no_run_id__")
-        batch_id = validation_result.batch_id
-        check_details_text = f"*{n_checks_succeeded}* of *{n_checks}* expectations were met"
-
-        failed_expectations_text = ""
-
         status = "Failed :x:"
         if validation_result.success:
             status = "Success :tada:"
-        else:  # noqa: PLR5501
-            if show_failed_expectations:
-                failed_expectations_text = self.create_failed_expectations_text(
-                    validation_result["results"]
-                )
 
+        validation_link = None
         summary_text = ""
         if validation_result_urls:
-            # This adds hyperlinks for defined URL
             if len(validation_result_urls) == 1:
-                title_hlink = f"*<{validation_result_urls[0]} | Validation Result>*"
+                validation_link = validation_result_urls[0]
             else:
-                title_hlink = "*Validation Result*"
-            batch_validation_status_hlinks = "".join(
-                f"*Batch Validation Status*: *<{validation_result_url} | {status}>*"
-                for validation_result_url in validation_result_urls
+                title_hlink = "*Validation Results*"
+                batch_validation_status_hlinks = "".join(
+                    f"*<{validation_result_url} | {status}>*"
+                    for validation_result_url in validation_result_urls
+                )
+                summary_text += f"""{title_hlink}
+    {batch_validation_status_hlinks}
+                """
+
+        expectation_suite_name = validation_result.suite_name
+        data_asset_name = validation_result.asset_name or "__no_data_asset_name__"
```

Comment:
I think we should always have an `asset_name` - we may want to remove the property we have at the top level of the `ExpectationSuiteValidationResult`, add it as a param to the constructor, and make sure that whoever is creating the object is providing the necessary data

Reply:
will address this as a separate thing because editing the  `ExpectationSuiteValidationResult` which will result in many TFs and code changes that I'd rather not make as part of this effort